go_modules/spec/dependabot/go_modules/file_updater/go_mod_updater_spec.rb

```diff
@@ -664,7 +670,9 @@ module github.com/dependabot/vgotest
       end
 
       context "with an unrestricted goprivate" do
-        let(:goprivate) { "" }
+        before do
+          ENV["GOPRIVATE"] = ""
+        end
```

Comment:
This is technically redundant, but I think it helps with clarity of the test so I left it in